pkg/web/static/js/envelope.js

```diff
@@ -1,43 +1,5 @@
-import { countriesArray, naturalPersonNtlIdTypeArray, networksArray } from './constants.js';
 import { setSuccessToast } from './utils.js';
 
-const network = 'network';
```

Comment:
This has been moved to a separate file so that this file will only execute code only for the send envelope form.